index.js

```diff
@@ -50,4 +50,4 @@ var crossTab = function (config) {
  * @param {*} Event data
  */
 
-module.exports = crossTab
+module.exports = { crossTab }
```

Comment:
Why was the export changed?

Reply:
@ai why switch to named export?

Reply:
Because webpack does not support default ESM export in some cases and also pack named exports much more efficient.

Reply:
Ok. Thank you for your explanation.